docker/testing/Dockerfile

```diff
@@ -37,3 +37,6 @@ RUN wget -O codecov https://codecov.io/bash && \
 # Install tarpaulin.
 RUN RUSTFLAGS="--cfg procmacro2_semver_exempt" \
     cargo install --version 0.9.1 cargo-tarpaulin
+
+# Install gocovmerge for e2e coverage.
+RUN GO111MODULE=on go get github.com/wadey/gocovmerge@b5bfa59ec0adc420475f97f89b58045c721d761c
```

Comment:
Maybe move the commit into an argument similar to other versions. Maybe it would also be useful if this was part of the development container?

Reply:
sure I'll do that